src/app/space/settings/collaborators/collaborators.component.ts

```diff
@@ -54,25 +56,28 @@ export class CollaboratorsComponent implements OnInit, OnDestroy {
     let pageSize = event.pageSize;
     console.log('event size from page', pageSize);
     pageSize = 20;
-    this.collaboratorSubscription = this.collaboratorService.getInitialBySpaceId(this.context.space.id, pageSize).subscribe(collaborators => {
+    this.subscriptions.push(this.collaboratorService.getInitialBySpaceId(this.context.space.id, pageSize).subscribe(collaborators => {
       this.collaborators = collaborators;
-    });
+      this.sortCollaborators();
+    }));
   }
 
   fetchMoreCollaborators($event): void {
-    this.collaboratorService.getNextCollaborators()
-      .subscribe(collaborators => {
-        if (collaborators) {
-          this.collaborators = this.collaborators.concat(collaborators);
-        }
-        }, err => {
-        console.log(err);
-      });
+    this.subscriptions.push(
+      this.collaboratorService.getNextCollaborators()
+        .subscribe(collaborators => {
+          if (collaborators) {
+            this.collaborators = this.collaborators.concat(collaborators);
+            this.sortCollaborators();
+          }
+          }, err => {
+          console.log(err);
```

Comment:
While we're here, can we incorporate the error handler/logging system instead of this? Alternatively, open an issue to track it and work on it later as it's low priority compared to other things at this time.

Reply:
Sure, done.